test/csv/test_headers.rb

```diff
@@ -315,4 +315,9 @@ def test_nil_row_header
     assert_equal([["A"], [nil]],
                  [row.headers, row.fields])
   end
+
+  def test_empty_string_value_with_headers_true
+    row = CSV.parse('', headers: true)
+    assert_equal([], row.headers)
```

Comment:
`assert_equal(CSV::Table.new([], []), table)` is better because it shows more information (such as class ans instance variables) on failure.

FYI: https://www.clear-code.com/blog/2011/2/28.html

Reply:
Thank you for information. Fixed it.